.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This codebase uses non-standard prettier style, adding this so it formats your code in way we want.

Reply:
Can't we use our eslint config?

Reply:
We can, but it's totally different style so I'd need to refactor entire codebase here (which doesn't really fit into this PR).